teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -209,14 +216,22 @@ The network interface(s) on which the node listens for P2P communication.
       hidden = true)
   private Integer minimumRandomlySelectedPeerCount;
 
+  @Option(
+      names = {"--p2p-static-peers-url"},
+      paramLabel = "<URL>",
+      description =
+          "Specifies a URL or file containing a list of 'static' peers (one per line) with which to establish and maintain connections",
```

Comment:
NIT: Maybe it is useful to clarify the format expected for the entries in the file (e.g. ENRs)?

Reply:
they're [multiaddr](https://docs.libp2p.io/concepts/appendix/glossary/#multiaddr) format - will fix comment